### PR TITLE
test(#1237): Resurrect `QnMergeTest`

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
1. Make `QnMergeTest` work
2. Add one more disabled test for #1237 (Verify CI checks before merge)

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
Blocked by #1627

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you have a look, please?

File: src/main/java/com/rultor/agents/github/qtn/QnMerge.java [hunk 1]
```diff
@@ -62,12 +62,13 @@ public final class QnMerge implements Question {
     /**
      * Message bundle.
      */
-    private static final ResourceBundle PHRASES =
+    public static final ResourceBundle PHRASES =
```
Comment (yegor256) [inline src/main/java/com/rultor/agents/github/qtn/QnMerge.java:65]: @volodya-lombrozo this is a bad idea

File: src/main/resources/phrases_en_US.properties [hunk 2]
```diff
@@ -32,6 +32,7 @@ QnMerge.start=OK, I'll try to merge now. You can check the progress of the merge
 QnMerge.already-closed=The pull request is closed already, so I can't merge it
 QnMerge.head-is-gone=Head repository is gone, can't merge from it
 QnMerge.base-is-gone=Base repository is gone, can't merge into it
+QnMerge.checks-are-failed=Can't merge it, some checks were failed.
```
Comment (yegor256) [inline src/main/resources/phrases_en_US.properties:35]: @volodya-lombrozo I would make a longer text here, with more details. Some people might be concerned about this situation and will think that the problem is in Rultor

**Review by yegor256 — CHANGES_REQUESTED** [review]
@volodya-lombrozo see a few comments

**Comment by volodya-lombrozo** [comment]
@yegor256 Reminder

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**